# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/linux/reporter/SandboxReporter.cpp
# Commit: ff9088972319
# Full Hash: ff9088972319ed49297b6cbd134e1d3fb121ed64
# Author: Jed Davis <jld@mozilla.com>
# Date: 2017-10-20 09:38:32
# Regressor Bug: 1328896
# File Overlap Count: 1
# Description:
#   Bug 1328896 - Restrict fcntl() in sandboxed content processes. r=gcp
#   
#   MozReview-Commit-ID: BDBTwlT82mf
# ==============================================================================

diff -r 36a034813888 -r ff9088972319 security/sandbox/linux/reporter/SandboxReporter.cpp
--- a/security/sandbox/linux/reporter/SandboxReporter.cpp	Fri Oct 20 00:19:58 2017 +0200
+++ b/security/sandbox/linux/reporter/SandboxReporter.cpp	Mon Jul 24 17:33:07 2017 -0600
@@ -200,6 +200,7 @@
     ARG_HEX(clone, 0); // flags
     ARG_DECIMAL(prctl, 0); // option
     ARG_HEX(ioctl, 1); // request
+    ARG_DECIMAL(fcntl, 1); // cmd
     ARG_DECIMAL(madvise, 2); // advice
     ARG_CLOCKID(clock_gettime, 0); // clk_id
 
